### DISCUSSION TITLE
5.3中引用了JsObject.h的模块都需要打开PCHUsage=NoPCHs吗？

### DISCUSSION THREAD

**Discussion by watsonsong** [Tencent/puerts#1569] [Q&A]
发现在5.3版本中，只要直接或者间接#include了v8的头文件，都需要打开NoPCHs。往往项目的主模块也会都在里面。
这样不会影响编译速度么，是否有什么办法可以避免呢？

**Comment by chexiongsheng** [comment]
貌似是在5.3下使用PCH的话，bEnableUndefinedIdentifierWarnings会不生效。

**Comment by BlurryLight** [comment]
1.  5.3 UBT似乎把bEnableUndefinedIdentifierWarnings给搞坏了，可以进UBT里手动修一下，禁用msvc的4668号warning就好了。
2.  实际测试下来，如果有incredibuild 这种联合编译工具的话，pch基本是负优化，关了pch反而会提速编译。
![image](https://github.com/Tencent/puerts/assets/14940338/f1f1e5ef-0e13-4ecf-89ec-c8b68cce91e9)
